src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/services/calendar/DueDateConfiguratorTest.java

```diff
@@ -327,12 +327,13 @@ public void shouldCalculateDateFromDueDateEvenWhenDueDateOriginPropertiesAreProv
 
     @ParameterizedTest
     @CsvSource(value = {
-        "true,true,6,6",
-        "true,true,8,8",
-        "true,false,6,6",
-        "false,true,6,6",
-        "false,true,2,2",
-        "false,false,6,6"
+        "true,Next,6,6",
+        "true,Next,8,8",
+        "true,No,6,6",
+        "false,Next,6,6",
+        "false,Next,2,2",
+        "false,Previous,2,2",
+        "false,No,6,6",
     })
```

Comment:
We appear to be missing scenarios 4 & 5 from AC when;

1. dueDateMustBeWorkingDay no provided in DMN or value not specified in DMN- A/C4 (MustBeWorkingDay not specified with a value or not specified as a parameter on DMN' - Default) 

2. Multiple dueDateMustBeWorkingDay - A/C5 (Multiple key value pairs for task for dueDateMustBeWorkingDay)